analyses/cell-type-ewings/scripts/README.md

```diff
@@ -239,5 +239,9 @@ The `utils` folder contains scripts with any helper functions that are used in `
 1. `jaccard-functions.R`: These functions are used to calculate the Jaccard similarity index between groups of cells.
    These functions are taken directly from https://github.com/AlexsLemonade/scpca-nf/blob/main/templates/qc_report/celltypes_supplemental_report.rmd.
 
-2. `tumor-validation-helpers.R`: These functions are used in the template notebooks in `template_notebooks/auc-workflow` and the notebooks found in the `exploratory_analysis/annotation_notebooks` folder.
+2. `tumor-validation-helpers.R`: These functions are used in the template notebooks in `template_notebooks/auc-singler-workflow` and the notebooks found in the `exploratory_analysis/annotation_notebooks` folder.
    The functions are helpful for exploring and validate tumor cell annotations in individual samples.
+   
+3. `clustering-functions.R`: These functions are used in the template notebook for exploring clustering, `template_notebooks/auc-singler-workflow/03-clustering.Rmd`. 
```

Comment:
If you remove functions from the exploratory notebook, also indicate here that the util script is used there too.